qml/pages/VenueList.qml

```diff
@@ -155,6 +155,7 @@ Page {
                     leftMargin: Theme.horizontalPageMargin
                     rightMargin: Theme.horizontalPageMargin
 
+                    topMargin: Theme.paddingSmall
```

Comment:
![virtualbox_sailfishos emulator_16_02_2017_08_22_27](https://cloud.githubusercontent.com/assets/11860038/23011444/4da86a2e-f421-11e6-89f7-dd75c8932ee6.png)

Look at the distance. That's how `development` currently (0b4dd4c) looks like.